src/org/feedhenry/Utils.groovy

```diff
@@ -36,3 +38,9 @@ def getArtifactsDir(name) {
 def gitRepoIsDirty(untrackedFiles='no') {
     return sh(returnStdout: true, script: "git status --porcelain --untracked-files=${untrackedFiles}").trim()
 }
+
+static Closure getDate() {
```

Comment:
Why is the return type Closure? Should it not be String since that's what SimpleDateFormat format method will return?